src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/Nodes/NodeTypes.cs

```diff
@@ -133,6 +133,7 @@ public enum NodeTypes
         SecurityPolicies,
         DatabaseScopedCredentials,
         ExternalTables,
+        DroppedLedgerTables,
```

Comment:
why adding here? it should be either alphabetically or add it to the end.

Reply:
I couldn't figure out the ordering from the given file, so I added it next to a similar item. Is there a preference between alphabetical or appending?

Reply:
it seems the values here are not sorted alphabetically, so in general, adding it to the end of the list is safer in case the int value is used somewhere.